src/features/chart/components/Chart.js

```diff
@@ -119,20 +120,27 @@ export const Chart = () => {
           </defs>
           <CartesianGrid opacity="0.2" />
           <XAxis
+            domain={["auto", "auto"]}
             dataKey="date"
+            tickFormatter={chartData}
```

Comment:
I don't think this is correct. It should be a function, not an array.

Reply:
oh, we don't need it anymore as we format the chartData already.
So, I can delete this line.